.changeset/metal-dragons-dream.md

```diff
@@ -0,0 +1,7 @@
+---
+"@mds-core/mds-policy-service": minor
+"@mds-core/mds-geography-service": patch
+"@mds-core/mds-types": patch
```

Comment:
For the `mds-types` change, might be worth splitting this into a separate changeset to say specifically that the change was fixing the `value_url` prop (could be an overly pedantic suggestion, though)

Reply:
if there was a larger change, rather than just additive, then perhaps. It's just monorepo.